llpc/context/llpcCompiler.cpp

```diff
@@ -795,6 +800,15 @@ Result Compiler::buildPipelineWithRelocatableElf(Context *context, ArrayRef<cons
 
   ElfPackage elf[ShaderStageNativeStageCount];
   assert(stageCacheAccesses.size() >= shaderInfo.size());
+
+  const MetroHash::Hash originalPipelineHash = context->getPipelineContext()->getPipelineHashCodeWithoutCompact();
+  LLPC_OUTS("LLPC version: " << format("0x%08" PRIx32, LLPC_INTERFACE_MAJOR_VERSION)
+                             << format("%08" PRIx32, LLPC_INTERFACE_MINOR_VERSION) << "\n");
+  LLPC_OUTS("Hash for pipeline cache lookup: " << format("0x%08" PRIx32, originalPipelineHash.dwords[0])
+                                               << format("%08" PRIx32, originalPipelineHash.dwords[1])
+                                               << format("%08" PRIx32, originalPipelineHash.dwords[2])
+                                               << format("%08" PRIx32, originalPipelineHash.dwords[3]) << "\n");
```

Comment:
Can you move this printing code to a helper function? You can also print more than 4 bytes at a time with `format_bytes`: https://llvm.org/doxygen/namespacellvm.html#ada8031029c6d5e40e89916058175dea3